addOns/network/src/main/java/org/zaproxy/addon/network/GlobalExclusionsOptions.java

```diff
@@ -186,8 +186,33 @@ protected void updateConfigsImpl(int fileVersion) {
                         ((HierarchicalConfiguration) getConfig()).configurationsAt(EXCLUSION_KEY);
                 if (fields.isEmpty()) {
                     setGlobalExclusions(DEFAULT_GLOBAL_EXCLUSIONS);
-                    break;
                 }
+                break;
```

Comment:
Why is this logic being changed?

Reply:
IDE complained about fall through, now that there is another case.

Reply:
It should fall through.

Reply:
Okay, I had to suppress the warning otherwise it wouldn't build.
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':addOns:network:compileJava'.
> Compilation failed; see the compiler output below.
  C:\Users\user\Desktop\zap-ws\zap-extensions\addOns\network\src\main\java\org\zaproxy\addon\network\GlobalExclusionsOptions.java:214: warning: [fallthrough] possible fall-through into case
              case 1:
              ^
  1 error
  1 warning
```